requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+flake8
```

Comment:
I tend to think it makes it easier for development, in the `install_requires` case. For `tests_require`, AFAIK that only works with `python setup.py test`, which we don't use here (and running `setup.py` is generally discouraged in the Python community).